src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs

```diff
@@ -120,8 +120,14 @@ public void Mutate(StrykerOptions options)
                         _logger.LogInformation("{0} mutants got status {1}", mutantsToSkip.Count(), MutantStatus.Skipped.ToString());
                     }
                 }
+                else
+                {
+                    _logger.LogError("Failed to restore the project to a buildable state. Please report the issue. Stryker can not proceed further");
```

Comment:
Can this be moved to the CompilingProcess class?

Reply:
I guess I can, but this sounds strange to me: it means that compilingprocess.Compile either succeed or throws.
What benefit do you see there?

Reply:
No